### ISSUE TITLE
custom "Bitrate" for avc1 h264 does not work

### ISSUE THREAD

**Issue by PostPollux** [MrKepzie/Natron#1227]
### Problem
WriteFFMpeg seems to be buggy with the "avc1 h264" codec.

**Expected behavior:** [What you expected to happen]
"Bitrate" under "Advanced" should have an impact on the filesize and quality of the video.

**Actual behavior:** [What actually happened]
I rendered a shot several times with avc1 h264 codec using different bitrates ranging from 1000 to 400000 and the result is always exactly the same! Same file size and same quality. 
I tested it with .mp4 and .mov and it's the same. Bitrate does not work.
 
### Versions

* **Natron version/commit:** 
Newest Linux Snapshot

* **OS version:** 
Linux Manjaro

**Comment by devernay** [comment]
"quality" is the right parameter to tune for H.264.
the default value for bitrate is 400000, with a tolerance of 40000000 (i.e. 400kb/s with a tolerance of 40Mb/s), which means it can be just about anything. Basically, one shouldn't touch these values for H.264 (except to raise even higher the tolerance)
if you set min/max quality to lower values, you will get the right effect. Can you confirm that it works for you?
I will check if I should remove the bitrate settings for H.264.
There is a recommendation in the "codec" choice to use preferably digital intermediate codecs (which H.264 is not, except if you use only I frames and a very low CRF setting), and encore with a separate tool.
I will put the same recommendation in the general help for the plugin.

**Comment by PostPollux** [comment]
Hi devernay!
Cool. Thanks for adjusting the visibility!
Ok, I tested it and changing the quality settings made a difference. After more then 5 renderings with guessing a good value I finally loaded it in Blender and bam, it worked like a charm. It was much faster and easier to adjust. And the result was exactly what I was expecting. This is not ment to be a complaint.

Things I noticed: 
- In Blender and in all other software I used so far there is always a "Bitrate" setting for the h264 codec which controls the quality. So I'm confused why quality is the right setting here. Are all the other softwares wrong on this point? And it's even more confusing which numbers to put in there.(low value = good quality)
- Blender seems to be much faster in the conversion and I think both use ffmpeg. What's the reason? (for Natron I used just a read and a write node) Is it because Natron loads everything in 32bits?
